src/inc/corinfo.h

```diff
@@ -2386,6 +2387,7 @@ class ICorStaticInfo
 
     virtual void getReadyToRunHelper(
             CORINFO_RESOLVED_TOKEN * pResolvedToken,
+            CORINFO_LOOKUP_KIND *    pLookupKind,
```

Comment:
Changed to just have a CORINFO_LOOKUP argument

---
In reply to: [60819591](https://github.com/dotnet/coreclr/pull/4519#discussion_r60819591) [](ancestors = 60819591)